src/Sylius/Bundle/CustomerBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,150 @@
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CustomerBundle\DependencyInjection;
+
+use Sylius\Bundle\ResourceBundle\Controller\ResourceController;
+use Sylius\Bundle\ResourceBundle\Form\Type\ResourceChoiceType;
+use Sylius\Bundle\ResourceBundle\SyliusResourceBundle;
+use Sylius\Bundle\CustomerBundle\Controller\CustomerController;
+use Sylius\Bundle\ResourceBundle\Form\Type\ResourceFromIdentifierType;
+use Sylius\Bundle\CustomerBundle\Form\Type\CustomerProfileType;
+use Sylius\Bundle\CustomerBundle\Form\Type\CustomerType;
+use Sylius\Bundle\CustomerBundle\Form\Type\GroupType;
+use Sylius\Component\Resource\Factory\Factory;
+use Sylius\Component\Customer\Model\Customer;
+use Sylius\Component\Customer\Model\CustomerInterface;
+use Sylius\Component\Customer\Model\Group;
+use Sylius\Component\Customer\Model\GroupInterface;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This class contains the configuration information for the bundle.
+ *
+ * This information is solely responsible for how the different configuration
+ * sections are normalized, and merged.
+ *
+ * @author Bartosz Siejka <bartosz.siejka@lakion.com>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('sylius_customer');
+
+        $rootNode
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
+        ;
+
+        $this->addResourcesSection($rootNode);
+
+        return $treeBuilder;
+    }
+
+    /**
+     * @param ArrayNodeDefinition $node
+     */
+    private function addResourcesSection(ArrayNodeDefinition $node)
+    {
+        $node
+            ->children()
+                ->arrayNode('resources')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('customer')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->variableNode('options')->end()
+                                ->arrayNode('classes')
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->scalarNode('model')->defaultValue(Customer::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('interface')->defaultValue(CustomerInterface::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('controller')->defaultValue(CustomerController::class)->cannotBeEmpty()->end()
```

Comment:
The `CustomerController` and corresponding routing should be removed from `CustomerBundle`